lighting/migrations/11.0.0.98.0/post-migration.py

```diff
@@ -0,0 +1,40 @@
+# Copyright NuoBiT Solutions - Kilian Niubo <kniubo@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from openupgradelib import openupgrade
+
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    if not version:
+        return
+    _logger.info("creating products...")
+
+    env.cr.execute("""alter table product_template add column x738374_lighting_product_id int4 not null""")
```

Comment:
Remove the "not null", it will throw an error if the table product_template already have records.